docs/Tutorials/Misc/DesktopApp.md

```diff
@@ -1,15 +1,15 @@
 # Desktop Application
 
-Normally in Pode you define a server and run it, however if you use the  [`Show-PodeGui`](../../../Functions/Core/Show-PodeGui) function Pode will serve the server up as a desktop application.
+Normally in Pode you define a server and run it, however if you use the [`Show-PodeGui`](../../../Functions/Core/Show-PodeGui) function Pode will serve the server up as a desktop application.
 
 !!! warning
-    Currently only supported in Windows PowerShell, and PowerShell 7 on Windows due to using WPF.
+Currently only supported in Windows PowerShell, and PowerShell 7 on Windows due to using WPF.
```

Comment:
The tab here is needed, for the warning to render properly.

Reply:
Sorry, didn't take that one away on intention, I guess that was a matter of my VSCodes MD Preview.